persim/pers_landscape_visual.py

```diff
@@ -0,0 +1,303 @@
+"""
+    Visualization methods for plotting persistence landscapes.
+
+    authors: Gabrielle Angeloro, Michael Catanzaro, Calder Sheagren
+"""
+
+import itertools
+from operator import itemgetter
+import matplotlib as mpl
+from matplotlib import cm
+import matplotlib.pyplot as plt
+import numpy as np
+from .pers_landscape import PersLandscape
+from .pers_landscape_exact import PersLandscapeExact
+from .pers_landscape_approx import PersLandscapeApprox
+
+mpl.rcParams["text.usetex"] = True
+
+__all__ = ['plot_landscape', 'plot_landscape_simple']
+
+
+def plot_landscape(
+    landscape: PersLandscape,
+    num_steps: int=3000,
+    colormap="default",
+    title=None,
+    labels=None,
+):
+    """
+    Plot landscape functions
+    """
+    if isinstance(landscape, PersLandscapeApprox):
+        return plot_landscape_approx(
+            landscape=landscape,
+            num_steps=num_steps,
+            colormap=colormap,
+            title=title,
+            labels=labels,
+        )
+    if isinstance(landscape, PersLandscapeExact):
+        return plot_landscape_exact(
+            landscape=landscape,
+            num_steps=num_steps,
+            colormap=colormap,
+            title=title,
+            labels=labels,
+        )
+
+
+def plot_landscape_simple(landscape: PersLandscape,
+                   alpha=1,
+                   padding=0.1,
+                   num_steps=1000,
+                   title=None,
+                   ):
+    """
+    plot landscape functions. 
+    """
+    if isinstance(landscape, PersLandscapeExact):
+        return plot_landscape_exact_simple(landscape=landscape, alpha=alpha, title=title)
+
+    if isinstance(landscape, PersLandscapeApprox):
+        return plot_landscape_approx_simple(landscape=landscape, alpha=alpha, padding=padding, num_steps=num_steps, title=title)
+
+
+def plot_landscape_exact(
+    landscape: PersLandscapeExact,
+    num_steps: int=3000,
+    colormap="default",
+    alpha=0.8,
+    labels=None,
+    padding: float=0.1,
+    depth_padding: float=0.7,
+    title=None,
+):
+    """
+    A plot of the exact persistence landscape.
```

Comment:
Have you tested how the documentation site looks with these docstrings?  They look good from here, but sometimes the docs render a bit wonky.

Reply:
Yes, but we can double check.